.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
Even though I think it's fine to add this, I would rather have it in a separate PR.

Reply:
Will remove it. And create separate PR.